### DISCUSSION TITLE
WRFHYDRO 5.2 - is it a stable version ?

### DISCUSSION THREAD

**Discussion by jbensabat** [NCAR/wrf_hydro_nwm_public#719] [General]
I have downloaded the software from GITHUB and I See that many options have been added and do not seem to be documented 
CROCUS 
additional variables in the soil_parameters.nc file
Is there any update on the doc ?
Also in some routines (Such as the NWM) so calls to crucial routines (loading of hydro,namelist) are commneted out 
please help 
best 
jac

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version of the code used:
* Operating System and version:
* Compiler and version:
* Other relevant information:

**Comment by scrasmussen** [comment]
Hi @jbensabat, 

- Here is [a paper](https://hess.copernicus.org/articles/25/4275/2021/hess-25-4275-2021.pdf) that describes Crocus in WRF-Hydro.
- You can look at the commit details of the `SOILPARM.TBL` file [here](https://github.com/NCAR/wrf_hydro_nwm_public/commits/57dce00284bb0411beec7b7d9008395266ea259f/trunk/NDHMS/Land_models/NoahMP/run/SOILPARM.TBL?browsing_rename_history=true&new_path=src/Land_models/NoahMP/run/SOILPARM.TBL&original_branch=main) to see how it has changed over time. You might be interested in the changes from the [Xinanjiang Surface Runoff Scheme](https://github.com/NCAR/wrf_hydro_nwm_public/commit/8a61c8fbf3d12505f558611dcb1cc5e4e9337f74) commit.
- If crucial routines are commented out, it would be leftovers from older code but the functionality would be replicated in some other area of the source code. As the project has grown restructuring of the codebase has had to take place at times.
